client/gutenberg/editor/style.scss

```diff
@@ -13,3 +13,19 @@
 .editor {
 	background: none;
 }
+
+// OVERRIDES
+
+.layout__content {
```

Comment:
Instead of `!important` I think we can use the fact that all of the bellow classes are nested inside of `is-section-gutenberg-editor`.

So we could just apply the rules to:
 - `.is-section-gutenberg-editor .layout__content`
 - `.is-section-gutenberg-editor .edit-post-header`
 - `.is-section-gutenberg-editor .edit-post-sidebar`

and remove all of the `!important` entries.